### DISCUSSION TITLE
Indexing Error

### DISCUSSION THREAD

**Discussion by picklerick323** [photoprism/photoprism#3782] [Q&A]
I have been getting this error in a couple of photos that I upload. I've attached a photo that gives me the error.

thumb: unrecognised image format in 20231001_134923_0D85D20D.jpg (read color metadata)
2023-10-02 16:31:25
index: failed creating thumbnails for 2023/10/20231001_134923_0D85D20D.jpg (image: unknown format while decoding)
2023-10-02 16:31:25

![20231001_134923_0D85D20D](https://github.com/photoprism/photoprism/assets/83799787/11bbb416-df2c-48f5-8fd6-bf775e736e0d)

**Comment by graciousgrey** [comment]
Thank you very much for reporting. 

@lastzero the logs additionally show:

`metadata: not on new segment marker @ (457856): (00) while parsing jpeg file`

When looking at the exif data I see at least one warning, not sure if it is related.

**Reply by lastzero** [reply]
Could these files actually be saved in a different format? See https://www.stellarinfo.com/blog/fix-photoshop-an-unknown-or-invalid-jpeg-marker-type-is-found-error/

**Reply by graciousgrey** [reply]
@lastzero I am not sure if it is related to the file type. If I remove the EXIF metadata completely, the file can be indexed without error.

**Reply by lastzero** [reply]
@graciousgrey This error may depend on the file type, as we use different Exif parsers depending on the image format. Could you please check what information (and what mime type) Exiftool displays for this file and create a GitHub Issue for looking into this if needed?

**Reply by lastzero** [reply]
An updated preview build with a fix is now available for testing:

https://docs.photoprism.app/getting-started/updates/#development-preview
